projects/pastanaga/src/lib/calendar/date-input.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="pa-date-input" 
+    [class.pa-date-error]="dateError"
+    [class.input-focused]="focused">
+    <pa-input [(ngModel)]="dateInput"
+        [placeholder]="datePlaceholder"
+        (valueChange)="checkTypedDate($event)"
+        (focus)="onFocus()"
+        (blur)="onBlur()"
+        [class.pa-field-label-error]="dateError"
+        [maxCharacters]="10"
+        id="date-input">
+        <ng-content></ng-content>
+    </pa-input>
+    <pa-button size="small" icon="event" [paPopup]="datepicker"
+    [class.pa-button-error]="dateError"></pa-button>
```

Comment:
I think it would be way easier to style the input if you used a `pa-input-icon` instead, which has a `iconClick` event you can listen to open the popup.